src/java/org/apache/cassandra/utils/memory/MemoryUtil.java

```diff
@@ -34,19 +34,20 @@
 
     private static final Unsafe unsafe;
     private static final Class<?> DIRECT_BYTE_BUFFER_CLASS, RO_DIRECT_BYTE_BUFFER_CLASS;
-    private static final long DIRECT_BYTE_BUFFER_ADDRESS_OFFSET;
```

Comment:
can you revert all changes?  as you migrated away from the heap buffer duplicate logic this logic is no longer needed to be changed